app/src/main/java/com/monkeyteam/chimpagne/ui/FindEventScreen.kt

```diff
@@ -511,7 +515,7 @@ fun FindEventMapScreen(
               modifier =
                   Modifier.padding(start = 12.dp, top = 12.dp)
                       .testTag("go_back")
-                      .shadow(elevation = 4.dp, shape = RoundedCornerShape(100))
+                      .shadow(elevation = 10.dp, shape = RoundedCornerShape(12.dp))
```

Comment:
10 is a lot for elevation. In the future, try to lowel the elevation

see the Official [Guidlines](https://designfornative.com/basics-of-elevation-on-android/l)

![image](https://github.com/Chimpagne/ChimpagneApp/assets/24866530/f56c7ee0-fd09-47c7-975d-8183d5fb8817)

Reply:
I changed this as requested :) Thank you for the input!